lms/static/js/student_account/views/AccessView.js

```diff
@@ -85,9 +86,14 @@
             },
 
             render: function() {
-                $(this.el).html( _.template(this.tpl)({
+                var output = HtmlUtils.template(this.tpl)({
                     mode: this.activeForm
-                }));
+                });
+
+                HtmlUtils.setHtml(
```

Comment:
Maybe I'm not understanding what this template returns, since I see you did that in other places. Why do you need to access ``output.text``?